### ISSUE TITLE
[Feature Request]: Add 2 and 4 weeks for timeframe of disappearing messages

### ISSUE THREAD

**Issue by RetroAbstract** [oxen-io/session-android#1013]
As the title says, this is a feature request to have the maximum timeframe for disappearing messages increased by adding the option of having messages disappear 2 weeks and 4 weeks after being sent.

**Comment by KeeJef** [comment]
In which case would you want this?

**Comment by bohwaz** [comment]
Yes, same need. We want to keep some groups for 2 weeks or 1 month. 1 week is too short.
